test/distributed/_tensor/test_pointwise_ops.py

```diff
@@ -268,5 +275,50 @@ def test_mul_out(self):
         self.assertEqual(expected, dt.to_local())
 
 
+class DistTensorRandomOpTest(DTensorTestBase):
```

Comment:
Can you add the test to a new file instead? pointwise ops tests aren't quite related to randomness